src/services/user/services/RegistrationLink.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable class-methods-use-this */
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+const logger = require('../../../logger');
+
+const { SC_SHORT_TITLE } = require('../../../../config/globals');
+
+const mailContent = (firstName, lastName, registrationLink) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['mailContent' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5448725)